mlflow/xgboost.py

```diff
@@ -0,0 +1,174 @@
+"""
+The ``mlflow.xgboost`` module provides an API for logging and loading XGBoost models.
+This module exports XGBoost models with the following flavors:
+
+XGBoost (native) format
+    This is the main flavor that can be loaded back into XGBoost.
+:py:mod:`mlflow.pyfunc`
+    Produced for use by generic pyfunc-based deployment tools and batch inference.
+"""
+
+from __future__ import absolute_import
+
+import os
+import yaml
+
+import mlflow
+from mlflow import pyfunc
+from mlflow.models import Model
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.utils.environment import _mlflow_conda_env
+from mlflow.utils.model_utils import _get_flavor_configuration
+
+FLAVOR_NAME = "xgboost"
+
+
+def get_default_conda_env():
+    """
+    :return: The default Conda environment for MLflow Models produced by calls to
+             :func:`save_model()` and :func:`log_model()`.
+    """
+    import xgboost as xgb
+
+    return _mlflow_conda_env(
+        additional_conda_deps=None,
+        additional_pip_deps=[
+            "xgboost=={}".format(xgb.__version__),
```

Comment:
Ah interesting, surprising that xgboost isn't available on the default conda channels :P (verified this by running `conda install xgboost` on my Mac, which failed)

Reply:
Did you try `conda install -c conda-forge xgboost`?
I always use this command to install xgboost on conda.

https://anaconda.org/conda-forge/xgboost